src/Grid.js

```diff
@@ -1,5 +1,30 @@
+import Tile from './Tile.js'
 export default class Grid {
-  constructor() {
+  constructor(size) {
+  	this.size = size;
+  	this.grid = [];
+  	while(this.grid.push([]) < size);

```

Comment:
Try something like this. This will do what you want.
`
this.grid = new Array(this.width);
    for (let i = 0; i < this.width; i++) {
      this.grid[i] = new Array(this.height);
    }
`
you can then initialize the values with `new Tile('0')`